research-hub-web/src/app/components/home/featured/featured.component.html

```diff
@@ -1,8 +1,9 @@
-<div *ngIf="featuredItems$ | async; let featuredItems">
-  <app-content-container>
-    <app-content-title [title]="title"></app-content-title>
-    <ng-container *ngIf="featuredItems.items.length > 0">
-      <app-cards [flex]="'row wrap'" [contentItem]="featuredItems"></app-cards>
-    </ng-container>
-  </app-content-container>
-</div>
+<app-content-container *ngIf="featuredItems$ | async as featuredItems">
```

Comment:
This looks much better! Just might be worth adding a bit of margin/gap between items as they are touching at certain screen widths (see at ~1236px)